tests/main/views/test_buyers.py

```diff
@@ -1922,21 +1922,41 @@ def test_show_draft_brief_summary_page(self, framework_status):
             )
         ]
 
-    @pytest.mark.parametrize('framework_status', ['live', 'expired'])
```

Comment:
We should keep coverage for both live and expired frameworks if we can - buyers do look at their old briefs, after all.

Reply:
This is about editing briefs though... can buyers edit their briefs from expired frameworks?

If a framework is expired, should we even show the tags? 🤔

Reply:
They can't edit them at all once they're published, regardless of framework status. But you can still evaluate a brief and update it with the outcome.

Reply:
Right. But I think this test is testing a brief that hasn't been published?

Reply:
So we need a new test?

Reply:
Yeah we need a new test